src/Components/test/testassets/BasicTestApp/MouseEventComponent.razor

```diff
@@ -43,49 +43,74 @@
     void OnMouseOver(MouseEventArgs e)
     {
         DumpEvent(e);
-        message += "onmouseover,";
+
+        if (e.Type == "mouseover")
```

Comment:
For these ones, couldn't we look at the dumped event? I imagine that's what these tests already do?

Reply:
Dumped events are dumped to the command line. I guess we can dump the `Type` specifically to another `message` like var and confirm it on the C# side. Updating...

Reply:
Hows this: https://github.com/dotnet/aspnetcore/pull/38792/commits/d40bae4deb7425936e70da135df13e5c2ed0878e